### ISSUE TITLE
Custom buttonText prop

### ISSUE THREAD

**Issue by jflayhart** [altayaydemir/react-dual-list-box#8]
Would be nice to allow custom button text or rather an icon that relies on font-awesome as a peer deps?

**Comment by jflayhart** [comment]
more like buttonIcon: https://github.com/altayaydemir/react-dual-list-box/pull/10

**Comment by altayaydemir** [comment]
Actually, I'm not a fan of adding peerDependencies to UI widgets like this. Instead of forcing to install font-awesome, we can use additional props like

```js
leftButtonContent: React.PropTypes.Node,
rightButtonContent: React.PropTypes.Node,
```

with defaultProps like
```js
leftButtonContent: '>'
rightButtonContent: '<'
```

**Comment by jflayhart** [comment]
Yeh, I had a feeling you might not be a fan of this and I agree this can complicate things as well as force users into using something they don't want.

How can we make this work so that either a text string is a valid option, while also allowing a configuration for someone like me to use some icon library like FA or Glyphicon?

**Comment by altayaydemir** [comment]
By using [React.PropTypes.Node](https://facebook.github.io/react/docs/typechecking-with-proptypes.html)

>   Anything that can be rendered: numbers, strings, elements or an array
  (or fragment) containing these types.

**Comment by altayaydemir** [comment]
```
const myIcon = (<fa class="icon" />);

const myBox = (
   <DualListBox
       leftButtonContent={myIcon}
   />
);
```

**Comment by altayaydemir** [comment]
Fixed in #10
